components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/TopSitesStorage.kt

```diff
@@ -41,8 +42,14 @@ interface TopSitesStorage : Observable<TopSitesStorage.Observer> {
      *
      * @param totalSites A total number of sites that will be retrieve if possible.
      * @param includeFrecent If true, includes frecent top site results.
+     * @param frecencyThreshold If [includeFrecent] is true, frecency threshold options for
+     * filtering visited sites based on their frecency score.
      */
-    suspend fun getTopSites(totalSites: Int, includeFrecent: Boolean): List<TopSite>
+    suspend fun getTopSites(
+        totalSites: Int,
+        includeFrecent: Boolean,
```

Comment:
Same comment re simplifying this down to a single `frecencyConfig: FrecencyThresholdOption?` value.

Reply:
Done